go-install/linux.md

```diff
@@ -1,21 +1,49 @@
-# Setup instructions for Linux
+# Personal Linux setup instructions
 
-1. Run the following commands in terminal:
+## Setup instructions for Ubuntu 20.04 LTS (focal)
```

Comment:
Might be worth putting a note saying that this only works on 20.04 and for other versions use the `other linux` guide

Reply:
Done